Unit-11/main.js

```diff
@@ -0,0 +1,97 @@
+// exercise-1
+const createElementMethodOne = (char,number) => {
+  const newArray = []
+  for(let i = 0;  i < number; i++) {
+    newArray.push(char)
+  }
+  return newArray
+}
+// exercise-1
+const createElementMethodTwo = (char,number) => {
+  const newArray = []
+  let i=0
+  while(i < number) {
+    newArray.push(char)
+    i+=1
+  }
+  return newArray
+}
+
+// exercise-2
+const createArrayReverse = (array = []) => {
+  const reverseArray = []
+  for(let i=array.length-1; i >= 0; i--) {
+    reverseArray.push(array[i])
+  }
+  return reverseArray
+}
+// exercise-3
+const deleteArray = (array = []) => {
+  const trueValues = []
+  for(let i = 0; i < array.length; i++) {
+    if(typeof(array[i]) == 'number' && array[i] !=0) {
```

Comment:
```suggestion
    if (typeof(array[i]) === 'number' && array[i] !== 0) {
```

Nhìn kỹ cách đặt dấu cách
dùng `===` và `!==` thay cho `==` và `!=`

Logic cũng đang sai:

Cơ bắp vậy làm gì khi đã có hàm filter?
Giải sai:

```javascript
const array = [0, 1, false, 2, undefined, '', 3, null, 'bé Công']
deleteFalseValue(array)
```
Text `bé Công` bay màu đi đâu rồi???